autoload/neomake.vim

```diff
@@ -531,6 +536,7 @@ function! s:Make(options) abort
         let s:make_id += 1
         let s:make_options[s:make_id] = {
                     \ 'cwd': getcwd(),
+                    \ 'verbosity': get(g:, 'neomake_verbose', 1) + &verbose,
```

Comment:
Well then, maybe it can be `4 - min([3, &verbose])` so that increasing the count makes it quieter?

Alternatively just make `:verbose` always cause the most verbose output regardless of the count?  Adding `:verbose` in front of the command is basically saying "tell me everything" in most of Vim's builtin commands.

In any case, I think having this feature makes `g:neomake_verbose` deprecated for users.  It would be more of a convenience for maker devs.  So, doing the addition isn't a terrible idea to me.